csrc/mmdeploy/apis/java/mmdeploy/Profiler.java

```diff
@@ -0,0 +1,23 @@
+package mmdeploy;
+
+public class Profiler {
+    static {
+        System.loadLibrary("mmdeploy_java");
+    }
+
+    public final long profilerHandle;
+    private String path_;
```

Comment:
Do we need to cache `path`?

Reply:
> Do we need to cache `path`?

I think if we want to analyse the profiler, we should use this path.

Reply:
`path` is already cached by C++ Profiler::Profiler